spec/features/user_sorts_todos_spec.rb

```diff
@@ -0,0 +1,103 @@
+require "rails_helper"
+
+feature ".index" do
+    before do
+        sign_in
```

Comment:
Nice work adding todos in the before block. Since we do a `sign_in` for every test in this file we can have a global `before` block that signs in